src/ogr/featuredefn.jl

```diff
@@ -330,7 +330,7 @@ function directly, but use OGRLayer::DeleteGeomField() instead.
 This method should only be called while there are no OGRFeature objects in
 existence based on this OGRFeatureDefn.
 """
-function deletegeomdefn!(featuredefn::FeatureDefn, i::Integer)::FeatureDefn
```

Comment:
I don't think it's very likely anyone is depending on the return type of a "delete!" function. I began annotating functions with their return types when I was vetting the entire package for consistency of function behavior with the corresponding names and it was difficult for me to analyze otherwise (even if just visually based on the function signature).